### PR TITLE
Fix, Update, and Unify clickstream

### PR DESCRIPTION
**Pull request by rmoff**
This updates the Clickstream demo, fixes an existing bug, and brings in the docker-compose version from the `ksql` repo.

### CODE REVIEW

**Comment by ybyzek** [comment]
You may need to modify the links referenced in the higher-level README: https://github.com/confluentinc/examples/blob/5.0.1-post/README.md

**Comment by ybyzek** [comment]
What's the difference between `docker-compose.yml` and `docker-compose-automatic.yml`?  There shouldn't be two docker-compose files, please consolidate.

File: clickstream/docker-compose-automatic.yml [hunk 1]
```diff
@@ -0,0 +1,209 @@
+---
+version: '2'
+services:
+  zookeeper:
```
Comment (ybyzek) [inline clickstream/docker-compose-automatic.yml:4]: This is where you could add container names so it shows up better for referencing in `docker-compose exec`, etc

**Review by ybyzek — CHANGES_REQUESTED** [review]
@rmoff : generally looks good, thanks for the PR!  The only issue that I strongly feel should be resolved before merge is the two docker compose files.

**Comment by rmoff** [comment]
> What's the difference between `docker-compose.yml` and `docker-compose-automatic.yml`? There shouldn't be two docker-compose files, please consolidate.

One is going to be used for the Product page, which this clickstream demo backs. I can exclude it from this PR if you'd like?

File: clickstream/docker-compose-automatic.yml (hunk 1 above)
Comment (rmoff) [reply clickstream/docker-compose-automatic.yml:4]: thanks, I'll do that in the future

**Review by JimGalasyn — APPROVED** [review]
LGTM!

**Comment by rmoff** [comment]
(sync'd with @ybyzek on Slack re. docker compose files)

File: clickstream/docker-compose-automatic.yml (hunk 1 above)
Comment (rmoff) [reply clickstream/docker-compose-automatic.yml:4]: Done: https://github.com/confluentinc/examples/pull/84/commits/22410d3864eaf7609ccefb181cf15af640243f8a

**Comment by rmoff** [comment]
Second Docker Compose file removed https://github.com/confluentinc/examples/pull/84

**Pull request merged**